src/RoslynPad.Hosting/ExecutionHost.cs

```diff
@@ -156,14 +159,18 @@ private static DelegatingTextWriter CreateConsoleWriter()
 
         public ExecutionHost(string hostPath, string initialWorkingDirectory,
```

Comment:
Perhaps I wasn't clear - I meant a class for all the parameters of the constructor. This way they would only be defined once instead of in several methods and fields.